test/WebSites/ValidationWebSite/bower.json

```diff
@@ -1,21 +0,0 @@
-{
-    "name": "ValidationWebSite",
-    "description": "Web site demonstrating various validations.",
-    "private": true,
-    "dependencies": {
-        "jquery": "1.11.2",
-        "jquery-validation": "1.13.1",
-        "jquery-validation-unobtrusive": "3.2.2"
```

Comment:
Not sure what you mean...I added these files in basic website now and you should be able to launch it from browser and test it too.